basic/subsumption.dat

```diff
@@ -18,10 +18,13 @@ arraysimple2.c		0
 arraysimple3.c		0
 arraysimple4.c		2
 arraysimple5.c		11
+boundCheck5		8
```

Comment:
@rasoolmaghareh This should now also be:
```
pointer_wp4.c 8
pointer_wp5.c 78
```
and preferably be moved right after `pointer_wp3.c ...`.